pmm/pmm-documentation.groovy

```diff
@@ -0,0 +1,51 @@
+pipeline {
+    agent {
```

Comment:
@puneet0191 Is this agent label needed? We're only rsyncing internally here.

Reply:
the agent you used, probably has the permission to rsync and ssh into the docs host, so we need it